dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/option/hibernate/OptionSet.hbm.xml

```diff
@@ -44,7 +44,7 @@
 
     <property name="publicAccess" length="8" />
 
-    <set name="userGroupAccesses" table="optionsetusergroupaccesses" cascade="all-delete-orphan">
+    <set name="userGroupAccesses" table="optionsetusergroupaccesses" cascade="all-delete-orphan" lazy="extra">
```

Comment:
What is lazy=extra doing and what's the need for it?

Reply:
Removed. That is to fix the lazy issue with caching. But i found another way to solve it.